styles.css

```diff
@@ -273,4 +273,126 @@
 }
 .alert-icon {
   margin-right: 6.25px;
+}
+
+
+/* Button component */
+.container__components-holder{
+  width: 90%;
+  height: fit-content;
+  margin-top: 2rem;
+
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: center;
+  /* background-color: rgba(101, 150, 27, 0.753) */
+}
+
+.link{
+  box-sizing:border-box;
+  text-decoration:none;
+  padding: 0.5rem 1rem;
+  
+}
+.link-primary{
+  background-color: rgba(0, 88, 189, 0.911);
```

Comment:
If you are re-using this color at multiple places, probably make a variable, otherwise this is fine.